pkg/relayer/proxy/server_builder.go

```diff
@@ -13,11 +14,22 @@ import (
 	suppliertypes "github.com/pokt-network/poktroll/x/supplier/types"
 )
 
-// supplierStakeWaitTime is the time to wait for the supplier to be staked before
-// attempting to retrieve the supplier's on-chain record.
-// This is useful for testing and development purposes, where the supplier
-// may not be staked before the relay miner starts.
-const supplierStakeWaitTime = 1
+const (
+	// supplierStakeWaitTime is the time to wait for the supplier to be staked before
+	// attempting to (try again to) retrieve the supplier's on-chain record.
+	// This is useful for testing and development purposes, where the supplier
+	// may not be staked before the relay miner starts.
+	supplierStakeWaitTime = 1
```

Comment:
There's no unit here mentioned in the comment nor in the variable name itself. 

I think it would improve readability to at least move the multiplication by `time.Second` here (instead of on line 168).